flow-nodes/api-builder-plugin-petstore/test.js

```diff
@@ -0,0 +1,74 @@
+const { expect } = require('chai');
+const { MockRuntime } = require('@axway/api-builder-sdk');
+
+const getPlugin = require('./index');
+
+function getLogger () {
+	const logger = {
+		debug: () => {},
+		trace: () => {},
+		info: () => {},
+		warn: () => {},
+		error: () => {},
+		fatal: () => {}
+	};
+	logger.scope = () => logger;
+	return logger;
+}
+
+describe('flow-node petstore', () => {
+	let runtime;
+	before(async () => {
+		const plugin = await getPlugin({}, {
+			logger: getLogger()
+		});
+		runtime = new MockRuntime(plugin);
+	});
+
+	describe('#constructor', () => {
+		it('should define flow-nodes', () => {
```

Comment:
We're not going to be able to scaffold these kinds of tests.  It means that the test effectively becomes: `getPlugin` and `verify`.  I wonder if there's a "neater" way to do this, and not have to expose the logger that way - maybe in the SDK.  e.g.:
```js
const { MockRuntime } = require('@axway/api-builder-sdk');
MockRuntime.loadPlugin('./index');
```

Reply:
yeah this is the ticket I mentioned in the last comment and on teams last week that I need to create. it may look slightly different, but there's a lot to do to simplify this stuff and not force the user to do stuff that API builder does. There does need to be a way of providing your own plugin config and API Builder config as well as stuff that the mockRuntime should provide by default